thermalnetwork/ground_heat_exchanger.py

```diff
@@ -66,7 +66,7 @@ def ghe_size(self, total_space_loads, output_path: Path) -> float:
             b_max=self.json_data["geometric_constraints"]["b_max"],
         )
         ghe.set_design(
-            flow_rate=self.json_data["design"]["flow_rate"], flow_type_str=self.json_data["design"]["flow_type"]
+            flow_rate=self.json_data["design"]["flow_rate"] * 1000, flow_type_str=self.json_data["design"]["flow_type"]
```

Comment:
Yikes, this seems like a big bugfix! 😆

I confirmed that [GHED is expecting lps](https://github.com/BETSRG/GHEDesigner/blob/main/ghedesigner/manager.py#L430), but wasn't able to confirm the units we're sending. It's [not defined in the GMT schema](https://github.com/urbanopt/geojson-modelica-translator/blob/develop/geojson_modelica_translator/system_parameters/schema.json#L1615-L1629).

Your change obviously addresses this quickly. It's not clear to me that it's required, though. If you say so, I'm good with it! I'll update the GMT schema to clarify that this is in m3/s. 

For the long term: I'd like to make an issue both here and in the GMT to change flow_rate to be in lps natively. We can see in the schema that flow_type is in lps, which seems like a typo.

Reply:
@vtnate I'm good with updating the GMT schema to define the flow to be in lps. That would negate this PR, which is the best path.